packages/sdk/wallets/wallet-ledger/tokens-all-networks.json

```diff
@@ -1,11 +1,21 @@
 [{
-  "ticker": "CELO",
-  "address": "471ece3750da237f93b8e339c536989b8978a438",
+  "ticker": "USD₮",
+  "address": "0E2A3e05bc9A16F5292A6170456A710cb89C6f72",
+  "decimals": 18,
+  "chainId": 42220
+},{
+  "ticker": "USDC",
+  "address": "2F25deB3848C207fc8E0c34035B3Ba7fC157602B",
+  "decimals": 6,
+  "chainId": 42220
+},{
+  "ticker": "eXOF",
+  "address": "73F93dcc49cB8A239e2032663e9475dd5ef29A08",
   "decimals": 18,
   "chainId": 42220
 },{
   "ticker": "cUSD",
-  "address": "765de816845861e75a25fca122bb6898b8b1282a",
+  "address": "765DE816845861e75A25fCA122bb6898B8B1282a",
```

Comment:
todo check the change in case is cool?

Reply:
It's the actual checksumed address gotten back from the whitelist contract call, so I believe this is more correct.